pipelines/h4c/post_processing/v7_interleave/post_lstbin_frate/task_scripts/do_EXTRACT_AUTOS.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+set -e
+
+#import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# This script extracts the autocorrelations into a single waterfall file
+label=${1}
+include_diffs=${2}
+
```

Comment:
It's possible that we'll need to manually move into the directory with the files. Figured I should flag it here as a reminder in case things break in the future and it's not obvious why.